examples/chainlink/chainlink.py

```diff
@@ -12,8 +12,28 @@
 class ChainlinkService(Service):
     @staticmethod
     async def get_payload():
-        return await vm.repo(
-            image_hash="bdcd10f2cc95e296174975d9a1a170ee0d0efd01b5ff8316aa9a65fa",
+        manifest = "eyJ2ZXJzaW9uIjoiMC4xLjAiLCJjcmVhdGVkQXQiOiIyMDIxLTExLTA5VDE5OjE5OjQxL" \
```

Comment:
I understand this manifest contains the image address or hash, right?

Reply:
That's a base64-encoded computation (app) manifest:

https://github.com/golemfactory/golem-architecture/blob/85d3e85817fbc29820310a93be83ad5375f8c067/gaps/gap-4_comp_manifest/gap-4_comp_manifest.md